File: C++/Programming Basics with C++/10. For-loop/simple_tasks/OddEvenSum.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    int n;
    cin >> n;
    int evenSum = 0;
    int oddSum = 0;
    for (int i = 0; i < n; i++) {
        int num;
        cin >> num;
        if (i % 2 == 0) {
            evenSum += num;
        } else {
            oddSum += num;
        }

    }

    if (evenSum == oddSum) {
        cout << "Yes, sum = " << evenSum << endl;
    } else {
        cout << "No, diff = " << abs(evenSum - oddSum) << endl;
    }
    return 0;
}

File: C#/Programming Basics with C#/10. For-Loop/07. Odd Even Sum/OddEvenSum.cs
using System;

class OddEvenSum
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        int sumOdd = 0;
        int sumEven = 0;
        for (int i = 1; i <= n; i++)
        {
            int num = int.Parse(Console.ReadLine());
            if (i % 2 == 0)
            {
                sumEven = sumEven + num;
            }
            else
            {
                sumOdd = sumOdd + num;
            }
        }

        if (sumOdd == sumEven)
        {
            Console.WriteLine("Yes");
            Console.WriteLine($"Sum = {sumOdd}");
        }
        else
        {
            Console.WriteLine("No");
            Console.WriteLine($"Diff = {Math.Abs(sumOdd - sumEven)}");
        }
    }
}